com.valvesoftware.Steam.yml

```diff
@@ -105,6 +105,8 @@ modules:
 
   - "modules/discord.yml"
 
+  - "modules/faudio.yml"
```

Comment:
I think it would be more obvious what the purpose is if the module was called proton.

Reply:
If you say so.